docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -177,6 +177,59 @@ a few caveats:
   - Enable using Apache Arrow serialization when reading data from BigQuery.
     Please read this [section](bigquery-arrow-serialization-support) before using this feature.
   - `true`
+* - `bigquery.channel-pool.initial-size`
+  - The initial size of the connection pool, also known as a channel pool,
+    used for gRPC communication.
```

Comment:
do we explain gRPC and RPC somewhere earlier in the doc? If not .. we might want to .. or at least link to some external doc

Reply:
Let's handle this topic in a follow-up.
The term gRPC is already mentioned in https://trino.io/docs/current/connector/bigquery.html#bigquery-storage-api